app/controllers/submit.js

```diff
@@ -506,22 +507,31 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
                     action: 'click',
                     label: 'Preprints - Submit - Save and Continue, Existing Node Existing File'
                 });
-            let node = this.get('node');
+
+            const node = this.get('node');
+            const currentTitle = node.get('title');
+            const nodeTitle = this.get('nodeTitle');
+
             this.set('basicsAbstract', this.get('node.description') || null);
 
-            if (node.get('title') !== this.get('nodeTitle')) {
-                let currentTitle = node.get('title');
-                node.set('title', this.get('nodeTitle'));
-                node.save()
-                    .then(() => this.get('abandonedPreprint') ? this.send('resumeAbandonedPreprint') : this.send('startPreprint'))
-                    .catch(() => {
-                        node.set('title', currentTitle);
-                        this.get('toast').error(this.get('i18n').t('submit.could_not_update_title'));
-                    });
+            // Ember.run(() => {
+            return Promise.resolve()
```

Comment:
It just starts a promise chain, because we need it for the return on L520 so that it's a 'thenable'. It could also be written as:

```js
(() => {
    if (currentTitle === nodeTitle) {
        return Promise.resolve();
    }

    node.set('title', nodeTitle);
    return node.save(); // returns a promise
})() // immediately invoked function
    .then( ... )
    .catch( ... )
```

Once you start the promise chain, you can just use return to go to the next .then without returning a value